column/api/run.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from flask import Flask
```

Comment:
Why Flask and not WSGI?  With wsgi we'll be able to have more options in how we stand up the service (apache2 or uwsgi, etc).  Also Flask’s built-in server is not suitable for production as it doesn’t scale well and by default serves only one request at a time.